cloudevents/formats/avro-turbo-format.md

```diff
@@ -0,0 +1,217 @@
+# Avro Turbo Event Format for CloudEvents - Version 1.0.3-wip
+
+## Abstract
+
+The Avro Turbo Format for CloudEvents defines how events attributes are expressed in
```

Comment:
I think this paragraph could do with rewriting - both here and in the original format - given that it's not just about attributes. It's how the whole event is represented in Avro, right? I'd also say it defines *one way* that CEs can be represented in Avro. Otherwise we've got two formats that say "This format defines how CEs are represented in Avro" and they can't both do that definitively.

Reply:
fixing